content/renderer/render_frame_impl.cc

```diff
@@ -2190,6 +2192,17 @@ void RenderFrameImpl::OnAddMessageToConsole(ConsoleMessageLevel level,
   AddMessageToConsole(level, message);
 }
 
+void RenderFrameImpl::OnInsertUserStyleSheetRequest(
```

Comment:
Would be good to add a reference here for the issue/ticket

Reply:
ACK